qiskit_experiments/library/calibration/drag.py

```diff
@@ -30,39 +30,41 @@
-    :math:`\beta` is the strength of the quadrature which we refer to as the DRAG
-    parameter and seek to calibrate in this experiment. The DRAG calibration will run
-    several series of circuits. In a given circuit a Rp(β) - Rm(β) block is repeated
-    :math:`N` times. Here, Rp is a rotation with a positive angle and Rm is the same rotation
-    with a native angle. As example the circuit of a single repetition, i.e. :math:`N=1`, is
-    shown below.
-
-    .. parsed-literal::
-
-                   ┌───────┐ ┌───────┐ ░ ┌─┐
-              q_0: ┤ Rp(β) ├─┤ Rm(β) ├─░─┤M├
-                   └───────┘ └───────┘ ░ └╥┘
-        measure: 1/═══════════════════════╩═
-                                          0
-
-    Here, the Rp gate and the Rm gate are can be pi and -pi rotations about the
-    x-axis of the Bloch sphere. The parameter β is scanned to find the value that minimizes
-    the leakage to the second excited state. Note that the analysis class requires this
-    experiment to run with three repetition numbers.
-
-    References:
+    # section: overview
+
+        A Derivative Removal by Adiabatic Gate (DRAG) pulse is designed to minimize leakage
+        to a neighbouring transition. It is a standard pulse with an additional derivative
+        component. It is designed to reduce the frequency spectrum of a normal pulse near
+        the :math:`|1\rangle` - :math:`|2\rangle` transition, reducing the chance of leakage
+        to the :math:`|2\rangle` state. The optimal value of the DRAG parameter is chosen to
+        minimize both leakage and phase errors resulting from the AC Stark shift.
+
+        .. math::
+
+            f(t) = \Omega(t) + 1j \beta d/dt \Omega(t)
+
+        Here, :math:`\Omega` is the envelop of the in-phase component of the pulse and
+        :math:`\beta` is the strength of the quadrature which we refer to as the DRAG
+        parameter and seek to calibrate in this experiment. The DRAG calibration will run
+        several series of circuits. In a given circuit a Rp(β) - Rm(β) block is repeated
+        :math:`N` times. Here, Rp is a rotation with a positive angle and Rm is the same rotation
+        with a native angle. As example the circuit of a single repetition, i.e. :math:`N=1`, is
+        shown below.
+
+        .. parsed-literal::
+
+                       ┌───────┐ ┌───────┐ ░ ┌─┐
+                  q_0: ┤ Rp(β) ├─┤ Rm(β) ├─░─┤M├
+                       └───────┘ └───────┘ ░ └╥┘
+            measure: 1/═══════════════════════╩═
+                                              0
+
+        Here, the Rp gate and the Rm gate are can be pi and -pi rotations about the
+        x-axis of the Bloch sphere. The parameter β is scanned to find the value that minimizes
+        the leakage to the second excited state. Note that the analysis class requires this
+        experiment to run with three repetition numbers.
+
+    # section: reference
```

Comment:
Would you mind replacing this block with

```
    .. ref_arxiv:: 1 1011.1949
    .. ref_arxiv:: 2 0901.0534
    .. ref_arxiv:: 3 1509.05470
```

? This is error free method for citation since string is automatically generated from the arxiv database, and the same formatting across all experiment documentation is guaranteed. Here, for example, citation 2 has no paper title. This kind of human error will not happen :)

Reply:
Cool feature! See here 0007b0b